### PR TITLE
Toggle coin balance

### PR DESCRIPTION
**Pull request by Kukushkina**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/ColuLocalNetwork/CLN-community-app/blob/master/docs/CONTRIBUTING.md)) doc
- [ ] Lint pass locally with my changes
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: client/src/components/CommunitiesList.js [hunk 1]
```diff
@@ -150,7 +152,8 @@ const mapStateToProps = state => {
   return {
     tokens: getCommunities(state),
     ui: state.ui,
-    selectedCommunity: getSelectedCommunity(state)
+    selectedCommunity: getSelectedCommunity(state),
+    account: state
```
Comment (leonprou) [inline client/src/components/CommunitiesList.js]: that's not account, it's the whole state. Also it's a bad practice to send the whole state to a component. correct way to use reselect - https://github.com/reduxjs/reselect#accessing-react-props-in-selectors

File: client/src/actions/ui.js [hunk 2]
```diff
@@ -30,6 +31,13 @@ export const hideModal = () => {
   }
 }
 
+export const toggleBalance = (coinBalance) => {
```
Comment (leonprou) [inline client/src/actions/ui.js]: rename this variable to showBalance. (it's  a bool, right?)

Comment (leonprou) [reply client/src/actions/ui.js]: @Kukushkina my bad, toggleBalance is good. I was referring to `coinBalance ` variable -> `showBalance`.  Cause it's a bool, right?

Comment (Kukushkina) [reply client/src/actions/ui.js]: Yes, it's boolean, I will revert to toggleBalance

**Pull request closed without merging**